### ISSUE TITLE
Display problem: disappearing initial letter

### ISSUE THREAD

**Issue by jcowey** [papyri/navigator#23]
The XML of the following line is correct:
https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/p.oxy/p.oxy.47/p.oxy.47.3338.xml#L79
in particular
`<expan>αἶγ<ex>ας</ex></expan>`
is correctly encoded. It is preceded by whitespace and before that
`<num value="110">ρ<supplied reason="lost">ι</supplied></num>`

When this is displayed in preview in the editor and in the navigator what we see in line 7 is:
`ρ[ι] ἶγ(ας)`
http://papyri.info/ddbdp/p.oxy;47;3338

In other words the initial alpha is missing.

I assume this must be related to an XSLT problem. It should be fixed. There have been some submissions to the board with attempts to correct this. It is usually the case that the underlying XML is correct.

**Comment by jcowey** [comment]
Now appears as required!!!
